content/pt-br/docs/reference/issues-security/security.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Informações de Segurança e Divulgação do Kubernetes
+aliases: [/security/]
+reviewers:
+- eparis
+- erictune
+- philips
+- jessfraz
+content_type: concept
+weight: 20
+---
+
+<!-- overview -->
+
+Esta página descreve informações de segurança e divulgação do Kubernetes.
+
+
+<!-- body -->
+## Anúncios de Segurança
+
+Junte-se ao grupo [kubernetes-security-announce](https://groups.google.com/forum/#!forum/kubernetes-security-announce) para receber e-mails sobre segurança e os principais anúncios de API.
+
+
+## Relatar uma Vulnerabilidade
+
+Somos extremamente gratos pelos desenvolvedores de segurança e usuários que relatam vulnerabilidades para a Comunidade Open Source do Kubernetes. Todos os relatos são minuciosamente investigados por um conjunto de voluntários da comunidade.
+
+Para relatar, envie sua vulnerabilidade para o [programa de recompensa de bug do Kubernetes](https://hackerone.com/kubernetes). Isso possibilita a triagem e o tratamento da vulnerabilidade com tempos de resposta padronizados. 
+
+Você também pode enviar um e-mail para uma lista privada [security@kubernetes.io](mailto:security@kubernetes.io) com os detalhes de segurança e os detalhes esperados para [todos os relatos de bugs do Kubernetes](https://github.com/kubernetes/kubernetes/blob/master/.github/ISSUE_TEMPLATE/bug-report.yaml).
+
+Você pode criptografar seu e-mail para esta lista usando as chaves GPG dos [membros do Comitê de Resposta de Segurança](https://git.k8s.io/security/README.md#product-security-committee-psc). A criptografia usando GPG NÃO é requerida para fazer a divulgação.
+
+### Quando Devo Relatar uma Vulnerabilidade?
+
+- Você acha que você descobriu uma potencial vulnerabilidade de segurança no Kubernetes
+- Você não tem certeza como a vulnerabilidade afeta o Kubernetes
+- Você acha que você descobriu uma vulnerabilidade em outro projeto que o Kubernetes depende
+    - Para projetos com seus próprios processos de relatar e divulgar vulnerabilidade, por favor informe-o diretamente lá
+
+
+### Quando Não Devo Relatar uma Vulnerabilidade?
+
+- Você precisa de ajuda para ajustar os componentes do Kubernetes para segurança
+- Você precisa de ajuda para aplicar as atualizações relacionada a segurança
+- Seu problema não é relacionado a segurança
+
+## Resposta de Vulnerabilidade de Segurança
+
+Cada relato é reconhecido e analisado pelos membros do Comitê de Resposta de Segurança com 3 dias úteis. Isso iniciará o [Processo de Liberação de Segurança](https://git.k8s.io/security/security-release-process.md#disclosures).
```

Comment:
Some bits here:
* In this context, "acknowledge" has more of a confirmation meaning to me than recognition. Perhaps "cada relato é confirmado e analisado" could be a better translation?
* "Release" has the meaning of an actual version release in this context if I'm not mistaken. Therefore, I think we could keep it as "release" (instead of translating).